ui/app/appointments/controllers/manage/list/appointmentsListViewController.js

```diff
@@ -36,6 +38,53 @@ angular.module('bahmni.appointments')
                 $scope.isFilterOpen = $stateParams.isFilterOpen;
             };
 
+            var autoRefreshStatus = true;
```

Comment:
@rajur7, Please move this ``var autoRefreshStatus = true;`` to top as this is the default value